packages/core/src/components/text/text.tsx

```diff
@@ -17,10 +17,13 @@
 import classNames from "classnames";
 import * as React from "react";
 
-import { Classes } from "../../common";
+import { Classes, mergeRefs } from "../../common";
 import { DISPLAYNAME_PREFIX, Props } from "../../common/props";
 
-export interface TextProps extends Props {
+export interface TextProps
+    extends Props,
+        React.RefAttributes<HTMLElement>,
+        Omit<React.HTMLAttributes<HTMLElement>, "title"> {
```

Comment:
Changed this to better match other `Prop` interfaces elsewhere in the code base.